tests/RequestTest.php

```diff
@@ -1033,6 +1033,92 @@ public function testGetRefererInvalid()
         $this->assertNull($request->getReferer());
     }
 
+    /**
+     * @expectedException        Brick\Http\Exception\HttpBadRequestException
+     * @expectedExceptionMessage Invalid protocol: invalid_protocol
+     */
+    public function testGetCurrentShouldReturnHttpBadRequestException()
+    {
+        $request = new Request();
+        $_SERVER['SERVER_PROTOCOL'] = 'invalid_protocol';
+        $request->getCurrent();
```

Comment:
`Request::getCurrent()` is a static method. Please remove the `new Request` line and use `Request::` here.